src/component/form.js

```diff
@@ -58,8 +59,6 @@ const subTextStyles = StyleSheet.create({
     lineHeight: font.lineHeightSub,
     color: color.greyText,
     textAlign: 'center',
-    paddingTop: 40,
-    paddingBottom: 40,
```

Comment:
I removed padding in from `FormSubText` so that we can use it more generically e.g. in our SeedVerifyView.